Bearsampp/Bearsampp
# Convert from jq 1.x to 3.6.x

to test:
run ant build-lite
open local host
verify all spinners work and that the dashboard properly displays.
